tomltree_write.go

```diff
@@ -111,7 +111,7 @@ func (t *Tree) writeTo(w io.Writer, indent, keyspace string, bytesCount int64) (
 			return bytesCount, err
 		}
 
-		kvRepr := fmt.Sprintf("%s%s = %s\n", indent, k, repr)
+		kvRepr := indent + k + " = " + repr + "\n"
```

Comment:
You could probably speed this up even further by creating a bytes.Buffer outside of the loop range, writing to it inside the for loop, then calling `w.Write(buf.Bytes())`, which would also skip the string conversion below this line.